### PR TITLE
Configurable SideBarMenuItem content

### PR DESCRIPTION
**Pull request by danji90**
Updates:

- Added body prop (element) to SideBarMenuItem
- If provided body overwrites the title (which is string only)
- Adds functionality to the sidebar items, since they can contain non-String content

### CODE REVIEW

File: src/components/SidebarMenuItem/SidebarMenuItem.js [hunk 1]
```diff
@@ -50,6 +55,7 @@ const defaultProps = {
   className: 'tm-sidebar-item',
   tabIndex: 0,
   accessKey: undefined,
+  body: undefined,
```
Comment (oterral) [inline src/components/SidebarMenuItem/SidebarMenuItem.js]: body  seems very similar to children property , why not using the existing `children` property to define the content of the SideBarMenuItem ?

File: src/components/SidebarMenuItem/SidebarMenuItem.test.js [hunk 2]
```diff
@@ -42,6 +43,19 @@ describe('SidebarMenuItem', () => {
...
   });
 
+  test('should match snapshot with ActionLink as body.', () => {
+    const component = renderer.create(
+      <SidebarMenuItem
+        title="Title"
+        icon={<FaGithub />}
+        body={<ActionLink onClick={() => {}}>Click Me!</ActionLink>}
+        onClick={() => {}}
+      />,
```
Comment (oterral) [inline src/components/SidebarMenuItem/SidebarMenuItem.test.js]: Using children prop we can define Item like this :  
```<SidebarMenuItem
        title="Title"
        icon={<FaGithub />}
        onClick={() => {}}
      >
      <ActionLink onClick={() => {}}>Click Me!</ActionLink>
</SidebarMenuItem>,
```

File: src/components/SidebarMenuItem/SidebarMenuItem.js (hunk 1 above)
Comment (danji90) [reply src/components/SidebarMenuItem/SidebarMenuItem.js]: Makes total sense, just updated and pushed

File: src/components/SidebarMenuItem/SidebarMenuItem.test.js [hunk 3]
```diff
@@ -42,6 +43,16 @@ describe('SidebarMenuItem', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  test('should match snapshot with ActionLink as body.', () => {
+    const component = renderer.create(
+      <SidebarMenuItem title="Title" icon={<FaGithub />} onClick={() => {}}>
+        {<ActionLink onClick={() => {}}>Click Me!</ActionLink>}
```
Comment (oterral) [inline src/components/SidebarMenuItem/SidebarMenuItem.test.js]:
```suggestion
        <ActionLink onClick={() => {}}>Click Me!</ActionLink>
```

**Comment by danji90** [comment]
Thanks, I missed them brackets, luckily your reviews are immaculate :)

**Pull request merged**